nilearn/decoding/space_net.py

```diff
@@ -12,6 +12,7 @@
 
 import collections
 import time
+from typing import ClassVar
```

Comment:
You are getting some code style errors on the import.

See here: https://results.pre-commit.ci/run/github/1235740/1734281156.IFMbzslZRvWBfnsXqlJd_Q

My suggestion to fix them.

Install tox: `pip install tox`

From the root of the repository run:

```
tox
```

This should autofix most issues that you can them commit and push.